docs/contributing/translations.md

```diff
@@ -0,0 +1,36 @@
+# Translations
+Wagtail uses Transifex to translate the content for the admin interface. Our goal is to ensure that Wagtail can be used by those who speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code.
+
+## Translation workflow 
+Translations from Transifex are only integrated into the Wagtail repository at the time of a new release. When a release is close to being ready there will be a RC (Release Candidate) for the upcoming version and the translations will be exported to Transifex.
+
+During this RC period, usually around two weeks, there will be a chance for all the translators to update and add new translations. We will also notify the `#translators` channel in the Watail slack group at this time.
+
+These new translations are imported into Wagtail for any subsequent RC and the final release.
+
+If translations reach a threshold of about 80% we communicate that this is an officially supported translation in the docs.
+
+## What is Transifex
+ Wagtail is localised (translated) using [Transifex](https://www.transifex.com/), a web platform that helps organisations coordinate translation projects.
+
+## To help out in translating wagtail here's what you will do
```

Comment:
Wagtail, capital W

Reply:
Alternatively, try to avoid too many references to 'Wagtail' within this page as a whole.

```suggestion
## How to contribute translations
```

```suggestion
## How to help out with translations
```

Remember, people have come to the Wagtail documentation, adding Wagtail to titles is often just noise.